seacatauth/openidconnect/handler/authorize.py

```diff
@@ -161,12 +162,22 @@ async def authentication_code_flow(
 			)
 		# TODO: Fail with error response if client authorization fails
 		except KeyError:
-			L.info("Client ID not found", struct_data={"client_id": client_id})
-			# return self.reply_with_authentication_error(request, request_parameters, "invalid_client_id")
-		except exceptions.InvalidClientSecret as e:
-			L.info(str(e), struct_data={"client_id": client_id})
-			# return self.reply_with_authentication_error(request, request_parameters, "unauthorized_client")
-		except exceptions.InvalidRedirectURI as e:
+			L.warning("Client ID not found", struct_data={"client_id": client_id})
+			return self.reply_with_authentication_error(
+				"invalid_client_id",
+				redirect_uri,
+				error_description="Invalid client_id",
+				state=state
+			)
+		except client_exceptions.InvalidClientSecret as e:
+			L.warning(str(e), struct_data={"client_id": client_id})
+			return self.reply_with_authentication_error(
+				"unauthorized_client",
```

Comment:
Please add info that this value is from a standard (link to the specs)

Reply:
I extracted these error type strings into the member variables of `openidconnect.utils.AuthErrorResponseCode` and supplemented links to corresponding RFC standard.